docs/upgrading_to_v5.0.md

```diff
@@ -2,9 +2,9 @@
 
 The v5.0 release of *bootstrap* is a backwards incompatible release.
 
-## Terraform Validator < `v0.6.0` no longer supported
```

Comment:
This file should be kept as is since its for an old release. Next release will be v6.0

Reply:
Reverted